src/diff.js

```diff
@@ -0,0 +1,40 @@
+module.exports = { diffCoverage }
+
+
+function diffCoverage(jsonObj1, jsonObj2) {
+    mapping1 = createSourceFileNameToDataMap(jsonObj1)
+    mapping2 = createSourceFileNameToDataMap(jsonObj2)
```

Comment:
This is good for now, going to make a note that maybe we should move this transformation to the converter, and the diff can just assume it's in this structure